src/app/schools/schoolsShared/schools.services.ts

```diff
@@ -0,0 +1,15 @@
+import {Injectable} from '@angular/core';
+import {School} from './school';
+import {EntityMapperService} from '../../entity/entity-mapper.service';
+import {Observable} from '../../../../node_modules/rxjs/Rx';
+
+@Injectable()
+export class SchoolsServices {
```

Comment:
Naming convention is to use "service" in singular (i.e. file name "schools.service.ts", class name "SchoolsService")